### ISSUE TITLE
We should use yarn instead of npm

### ISSUE THREAD

**Issue by ms10398** [fossasia/susi_desktop#107]
**Actual Behaviour**

Instead of npm we should use yarn. As its new and better.

**Expected Behaviour**

Yarn should be used as the package manager in the app.

##### Please check the applicable fields -

- [x] I ensure that this issue is not a duplicate of any current open issues(s).
- [x] Assign this issue to me (I want to work on this).
- [x] I will send my PR only after I'm assigned to this issue by any maintainer.
- [x] I understand that I will be unassigned from this issue if I fail to send a PR for this within a week.

@anshumanv need your views for it.

Will bundle up everything in a single PR if you give a thumbsup to do it.

**Comment by anshumanv** [comment]
Sure we can use yarn, it's trending now and has some cool features, please do the needful! :smile:
